docs/plugins/argocdapp_plugin.md

```diff
@@ -18,8 +20,8 @@ tools:
     # version of the plugin
     # checkout the version from the GitHub releases
     version: 0.2.0
-  # optional; if specified, dtm will make sure the dependency is applied first before handling this tool.
-  dependsOn: [ "TOOL1_NAME.TOOL1_KIND", "TOOL2_NAME.TOOL2_KIND" ]
+  # if specified, dtm will make sure the dependency is applied first before handling this tool.
+  dependsOn: [ "ARGOCD_INSTANCE_NAME.argocd" ]
```

Comment:
INSTANCE -> TOOL? Because we call it TOOL_NAME across the doc/code.

Reply:
I think the ARGOCD_INSTANCE_NAME == TOOL_NAME, but ARGOCD_TOOL_NAME is not very good.
But TOOL_NAME is a little unclear.

Reply:
"Instance" means something specific. For example, in a highly-available setup, an "instance" might mean one of the ArgoCD server pods. So, "ARGOCD_INSTANCE_NAME" has ambiguity. Please think of a better name. If not possible, add some description about it, either in the example YAML as a comment, or as an extra sentence in the doc.